src/planeta/plasma-utils.js

```diff
@@ -15,6 +15,12 @@ class PlasmaMethodCall {
     // https://github.com/leapdao/leap-node/issues/298
     // ¯\_(ツ)_/¯
 
+    if(Array.isArray(privateKey)) {
```

Comment:
Could be skipped, but if it was me - I would move this into a helper inside utils :)

Reply:
Not sure why you would skip this :thinking: 

Maybe my intent is not clear. To send a transaction you need to sign (some of) its inputs. Sometimes you want to sign all of them, some other times only some of them. For Planet A we want to sign only the input that represents the player's passport and—in case of defect—an input with some Göllars.

To cover both use cases, `privateKey` can be an array or a string.

_I just noticed I need to remove the comment up there related to the leapdao/leap-node#298_

Reply:
Bad wording from me - I meant, that you can skip my suggestion :)